### PR TITLE
Manage/patient search

### PR DESCRIPTION
**Pull request by shimulH**

### CODE REVIEW

File: client/src/screens/Client/Manage/AccountingSearch/AccountingSearch.js [hunk 1]
```diff
@@ -203,23 +202,35 @@ export default function AccountingSearch() {
...
               </Grid>
               <Grid item xs={12} sm={6}>
-                <TextField
+                <KeyboardDatePicker
                   variant="outlined"
                   id="date"
                   label="Date From"
                   value={dateFrom}
                   className={classes.textField}
                   onChange={handleDateChangeFrom}
```
Comment (avastamin) [inline client/src/screens/Client/Manage/AccountingSearch/AccountingSearch.js:211]: You don't need separate function. You can do in the following way: `onChange={(date)=> setDateFrom(date)}`

File: client/src/screens/Client/Manage/AccountingSearch/AccountingSearch.js [hunk 2]
```diff
@@ -203,23 +202,35 @@ export default function AccountingSearch() {
...
+                  }}
+                  format="yyyy/MM/dd"
+                  inputVariant="outlined"
                   variant="outlined"
                   id="date"
                   label="Date To"
-                  type="date"
                   value={dateTo}
                   className={classes.textField}
                   onChange={handleDateChangeTo}
```
Comment (avastamin) [inline client/src/screens/Client/Manage/AccountingSearch/AccountingSearch.js:236]: `onChange={(date)=> setDateTo(date)}`

File: client/src/screens/Client/Manage/PatientSearch/PatientSearch.js [hunk 3]
```diff
@@ -10,23 +22,373 @@ const useStyles = makeStyles((theme) => ({
...
+                    KeyboardButtonProps={{
+                      "aria-label": "change date",
+                    }}
+                    format="yyyy/MM/dd"
+                    inputVariant="outlined"
+                    id="createdFrom"
+                    label="Created From"
+                    value={createdFrom}
+                    className={classes.textField}
+                    onChange={(date) => setCreatedFrom(date)}
```
Comment (avastamin) [inline client/src/screens/Client/Manage/PatientSearch/PatientSearch.js:181]: Same here...

File: client/src/screens/Client/Manage/PatientSearch/PatientSearch.js [hunk 4]
```diff
@@ -10,23 +22,373 @@ const useStyles = makeStyles((theme) => ({
...
+                    value={phone}
+                    variant="outlined"
+                    margin="normal"
+                    name="phone"
+                    label="Phone"
+                    size="small"
+                    type="phone"
+                    id="phone"
+                    autoComplete="phone"
+                    onChange={(event) => setPhone(event.target.value)}
```
Comment (avastamin) [inline client/src/screens/Client/Manage/PatientSearch/PatientSearch.js:218]: 👍

File: client/src/screens/Client/Manage/PatientSearch/PatientSearch.js [hunk 5]
```diff
@@ -10,23 +22,373 @@ const useStyles = makeStyles((theme) => ({
...
+                    className={classes.textField}
+                    value={appointmentTo}
+                    variant="outlined"
+                    margin="normal"
+                    name="appointmentTO"
+                    size="small"
+                    label="Appointment To"
+                    id="appointmentTO"
+                    autoComplete="appointmentTO"
+                    onChange={(date) => setAppointmentTO(date)}
```
Comment (avastamin) [inline client/src/screens/Client/Manage/PatientSearch/PatientSearch.js:275]: Same here...

**Review by avastamin — APPROVED** [review]

**Comment by shimulH** [comment]
Did all the changes

**Pull request merged**